matching-game/tests/components/button.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import { Button } from '@/components/button/button'
+
+describe('Button', () => {
+  it('Renders the correct text', () => {
+    render(<Button text="Testing" />)
+    expect(screen.getByTestId('button')).toHaveTextContent('Testing')
```

Comment:
then, when that button is rendered, we make an assertion on that. in this case, i expect the element with `data-testid=button` to have text `Testing`.